shablona/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .shablona import *
-from .version import __version__
\ No newline at end of file
+from .shablona import *  # noqa
```

Comment:
Just noticed this: would you mind explaining (here, or elsewhere) what "noqa" means? I assume this is something that flake8 recognizes?